ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/flow-tracking.yml

```diff
@@ -12,6 +12,8 @@ flow_trackings:
           mpls: true
         # no exporter
       - name: T2
+        # table size as string
+        table_size: "999999"
```

Comment:
I have read the close comments about the upper limit being platform dependent.
However can we use in the molecule test some values that would be accepted on vEOS / cEOS in case we start trying the configs on such devices:

vEOS 4.27.0F output:
```
Spine1(config-ftr-sampled-tr-T2)#flow table size ?
  <1-614400>  Maximum number of entries in flow table
```